README.md

```diff
@@ -85,4 +95,4 @@ jobs:
 ]
 ```
 
-From this repository usage https://github.com/EddieJaoudeCommunity/awesome-github-profiles
+From this repository usage <https://github.com/EddieJaoudeCommunity/awesome-github-profiles>
```

Comment:
I think this is not needed

Reply:
This is to fix Markdown linter issues. 🙂
I can still remove it if you want, though.

Reply:
Probably best to have it as a raw link or use a markdown link but not inbetween, I will merge and another PR can fix this

Reply:
I'll submit a good first issue for that later. Thanks Eddie!

Reply:
thanks 👍

Reply:
Looks good 👍